int/api/websites/all.go

```diff
@@ -19,7 +19,7 @@ import (
 const (
 	dateFormat          = "2006-01-02"
 	ownedPrefix         = "owned"
-	ownerKey            = "owner"
+	ownerKey            = "OWNER"
```

Comment:
maybe we should add a comment here to refer where we declare those keys on SC level ? what do you think ?